dialogue-core/src/main/java/com/palantir/dialogue/core/QueuedChannel.java

```diff
@@ -62,6 +63,7 @@
  */
 final class QueuedChannel implements Channel {
     private static final Logger log = LoggerFactory.getLogger(QueuedChannel.class);
+    private static final SkipLimits DO_NOT_SKIP_LIMITS = SkipLimits.No;
```

Comment:
Why not reference the enum directly?

Reply:
I was just trying to encode that really both codepaths here should use the same value, cause otherwise it's weird.